mcism_server/mcism_mcis.go

```diff
@@ -30,20 +31,40 @@ import (
 // Structs for REST API
 type vmReq struct {
 	Id             string `json:"id"`
-	Csp            string `json:"csp"`
-	Vm_image_name  string `json:"vm_image_name"`
-	Placement_algo string `json:"placement_algo"`
-	Name           string `json:"name"`
-	Description    string `json:"description"`
-
+	ConnectionName string `json:"connectionName"`
+
+	// 1. Required by CB-Spider
+	CspVmName string `json:"cspVmName"`
+
+	CspImageName          string   `json:"cspImageName"`
+	CspVirtualNetworkId   string   `json:"cspVirtualNetworkId"`
+	CspNetworkInterfaceId string   `json:"cspNetworkInterfaceId"`
+	CspPublicIPId         string   `json:"cspPublicIPId"`
+	CspSecurityGroupIds   []string `json:"cspSecurityGroupIds"`
+	CspSpecId             string   `json:"cspSpecId"`
+	CspKeyPairName        string   `json:"cspKeyPairName"`
```

Comment:
현재 API 컨밴션은 아래와 같은 스타일이었던 것 같습니다. JSON Key로 소대문자 구분 x, "_" 사용
spider에서 대소문자로 하고 있는 것 같은데, 우리도 변경이 필요할까요? 변경을 하게 되는 경우 다른 소스에도 영향을 많이 줄 것으로 보입니다.

```suggestion
	Csp_key_pair_name        string   `json:"csp_key_pair_name"`
```

Reply:
해당 변수명들은 Go 변수명이면서 JSON 변수명이기도 합니다.

1. JSON (또는 JavaScript) var name convention 으로 찾아 보면
강한 방침이 있는 것은 아닌 것 같고
예를 들면 https://www.quora.com/Should-I-use-camelCase-or-underscore-for-JSON 이 글에서는
JSON 을 활용할 언어 / 프레임워크 / 기술 스택에 따라 결정하면 될 것이라는 이야기들이 있습니다.

2. Go 에서는 주로 camelCase 를 쓰는 것으로 보입니다.
https://medium.com/@lynzt/variable-naming-conventions-in-go-89fe1ef17b0a
https://github.com/unknwon/go-code-convention/blob/master/en-US/naming_rules.md
https://golang.org/doc/effective_go.html